projects/reservoir/fv3/era_process.py

```diff
@@ -0,0 +1,213 @@
+import numpy as np
+import vcm
+import vcm.catalog
+from vcm.catalog import catalog
+import xarray as xr
+import netCDF4 as nc
+import glob
+import cftime
+import argparse
+import logging
+import os
+
+FREGRID_EXAMPLE_SOURCE_DATA = (
+    "gs://vcm-ml-raw/2020-11-12-gridspec-orography-and-mosaic-data/C48/*.nc"
+)
+
+var_id_mapping = {"sst": "sst", "temp2m": "t2m", "u_wind": "u10", "v_wind": "v10"}
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+def add_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--in_path", help="path to directory /downloaded which contains raw data"
```

Comment:
It's `<path>/var/downloaded`, right? As in the `in_path` arg should not contain the variable subdirectory?